controller/WebController.php

```diff
@@ -245,46 +243,43 @@ private function createFeedbackHeaders($fromName, $fromEmail, $toMail, $sender)
         if (!empty($fromEmail)) {
             $headers .= "Reply-To: $fromName <$fromEmail>\r\n";
         }
-
         $service = $this->model->getConfig()->getServiceName();
         return $headers . "From: $fromName via $service <$sender>";
     }
 
     /**
      * Sends the user entered message through the php's mailer.
-     * @param string $message only required parameter is the actual message.
+     * @param string $message content given by user.
+     * @param string $messageSubject subject line given by user.
      * @param string $fromName senders own name.
      * @param string $fromEmail senders email address.
      * @param string $fromVocab which vocabulary is the feedback related to.
      */
-    public function sendFeedback($request, $message, $fromName = null, $fromEmail = null, $fromVocab = null, $toMail = null)
+    public function sendFeedback($request, $message, $messageSubject, $fromName = null, $fromEmail = null, $fromVocab = null, $toMail = null)
     {
         $toAddress = ($toMail) ? $toMail : $this->model->getConfig()->getFeedbackAddress();
+        $messageSubject = "[" . $this->model->getConfig()->getServiceName() . "] " . $messageSubject;
         if ($fromVocab !== null && $fromVocab !== '') {
             $message = 'Feedback from vocab: ' . strtoupper($fromVocab) . "<br />" . $message;
         }
-
```

Comment:
Some of these lines may have been there for comprehensibility. It is not a good idea to remove them out of no reason. If one does want to do such operations, it is recommended to be done in a separate commit/branch in order to not confuse the PR.